LICENSE.md

```diff
@@ -179,7 +179,7 @@ recommend that a file or class name and description of purpose be included on
 the same “printed page” as the copyright notice for easier identification within
 third-party archives.
 
-    Copyright 2021 F. Hoffmann-La Roche AG and GlaxoSmithKline LLC
+    Copyright 2024 F. Hoffmann-La Roche AG, GlaxoSmithKline LLC, Pfizer and Cytel
```

Comment:
```suggestion
    Copyright 2024 F. Hoffmann-La Roche AG, GSK plc, Pfizer and Cytel
```

Totally forgot but I recall they changed their name and moving forward it should be this?

Reply:
A quick Google seems to say you are correct 😄 FYI @bms63 in case this needs to be updated in any other LICENSE files.

Reply:
This was discussed a while back, I think with @fshanlee, and it was decided to leave as GlaxoSmithKline LLC, as this is the legal entity??

Reply:
Maybe @MichaelRimler also discussed this previously??

Reply:
ah ok, let me put back how it was and can always update later as you need